Dockerfile

```diff
@@ -10,7 +10,7 @@
 ## in relation to the total image size.
 ## By choosing a very basic base image, we do keep full control over every part
 ## of the build steps. This Dockerfile contains every piece of magic we want.
-FROM rust:1.54-slim-buster AS builder
+FROM rust:1.54-slim-bullseye AS builder
```

Comment:
It's a bit of convenience; this allows us to skip the `rust` installation altogether.
As the [`rust` Docker Image ](https://hub.docker.com/_/rust) is an official image based upon Debian, I thought it would be the simplest to base our work on top of that image.

Reply:
Maybe, there is no difference technically, but:
* it's makes our docker image declaration intent a bit vague (like we are based on rust, or heavily dependent on it)
* this way of pining version can be used only once - and can't be used for some other stuff. (*)

(*) unless we are planning to move to multi-staged builds :partying_face: 
https://github.com/timescale/timescaledb-docker-ha/blob/be021d29b0a8b8f0d0f87a37ee413e32400a2788/Dockerfile#L313 - or -- we already using them?

Reply:
Yes, we're already using multi-stage builds, however not effeciently yet.

https://github.com/timescale/timescaledb-docker-ha/pull/157 is close to be ready; we use the multi-stages mostly to have a `--cache-from` for faster CI/CD runs for PR's for now.

> intent a bit vague
True, we could change it to base us upon vanilla debian again, we'd then need to include the stuff that rust adds.

Looking at the Dockerfile of the rust image, it seems we may easily incorporate that into our own, to remove the vague bit.
https://github.com/rust-lang/docker-rust/blob/21171fdd92e29acb045a41cd58b0d30d66aeaa7f/1.54.0/bullseye/Dockerfile

> or heavily dependent on it

Currently, we are heavily dependent on many compilers, for `c` they are taken from the `debian` distribution.

We could take `rustc`/`cargo` from the distribution though, for `bullseye` that would come down to `rustc` 1.48 currently.
For those projects that are being built in this image, they themselves build against rust 1.53/1.54.

Reply:
Ok, thanks for the clarification! Resolving this conversation, as it's not a blocker to merge (and never was).